Translation corrections (#270)
07th-mod/higurashi-console-arcs: Update/yoig_011.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>魅音</color>", NULL, "<color=#5ec69a>Mion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 43, "ps3/s10/43/410400642", 256, TRUE);
	OutputLine(NULL, "「あとは十和田が、それを力の限り後悔して、それからどうすればいいのか考えるのさ。",

> 			NULL, "\"At this point, Towada needs seriously regret what she's done, and think hard about what she should do from here on.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"At this point, Towada needs to seriously regret what she's done, and think hard about what she should do from here on.", Line_WaitForInput);
```

			NULL, "\"But... there's no longer room in there to lie to herself.", Line_WaitForInput);
	ModPlayVoiceLS(4, 43, "ps3/s10/43/410400650", 256, TRUE);
	OutputLine(NULL, "本当に正しい、…自分にとって一番の選択になると、私は信じている」",

> 			NULL, " So I believe... she will make the right choice... the choice best for herself.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " So I believe... she will make the right choice... the best choice for herself.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#e8af52>乙部</color>", NULL, "<color=#e8af52>Otobe</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 41, "ps3/s10/41/410200845", 256, TRUE);
	OutputLine(NULL, "「そうやって僕は、人のせいばかりにして、生きてきました…」",

> 			NULL, "\"I kept living on by blaming others for all my problems like that...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"I continued to go on living by blaming others for all my problems like that...\"", Line_Normal);
```

//r…そもそも、確実に、…なんてことを考える自体、傲慢で臆病な生き方だと、今ならわかる。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…そもそも、確実に、…なんてことを考える自体、傲慢で臆病な生き方だと、今ならわかる。",
			NULL, "...In hindsight... he realized even the act of thinking only in terms of successes was an arrogant and cowardly way of living.", Line_Normal);
	ClearMessage();

//r確実じゃないから、必死に力を振り絞って努力する。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "確実じゃないから、必死に力を振り絞って努力する。",

> 			NULL, "Mustering all your strength on something and trying your best because the result is uncertain.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Mustering all your strength on something and trying your best when the result is uncertain.", Line_Normal);
```

	ClearMessage();

//r限界なんてわからないから、納得するまであきらめない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "限界なんてわからないから、納得するまであきらめない。",
			NULL, "Not giving up until you discover your own limits.", Line_Normal);
	ClearMessage();

//rそうしなければ、本当の夢や適性なんて、わかるはずがない…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そうしなければ、本当の夢や適性なんて、わかるはずがない…。",

> 			NULL, "One cannot possibly hope to discover their true dreams and capabilities without putting in those efforts...", Line_Normal);

Reviewer: ```suggestion
			NULL, "One cannot possibly hope to discover their true dreams and capabilities without putting in that effort...", Line_Normal);
```

			NULL, "(But... pain is a natural part of life.", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s10/00/vt3a_chia1053", 256, TRUE);
	OutputLine(NULL, "…それを、どんなふうにして乗り越えていったかでしか…、人生の価値なんて計れないもの…。",

> 			NULL, " ...And the value of human life... can only be measured by the means in which we overcome that pain...", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " ...And the value of human life... can only be measured by the means by which we overcome that pain...", Line_WaitForInput);
```
